code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -522,10 +522,6 @@
 	list_reagents = list(/datum/reagent/consumable/lemon_lime = 30)
 	foodtype = FRUIT
 
-/obj/item/reagent_containers/food/drinks/soda_cans/lemon_lime/Initialize(mapload)
-	. = ..()
-	name = "lemon-lime soda"
```

Comment:
move this up to the actual name if you're removing this part

Reply:
oh, this was a mistake